pages/linux/genfstab.md

```diff
@@ -0,0 +1,20 @@
+# genfstab
+
+> Generate output suitable for addition to an fstab file.
+> More information: <https://man.archlinux.org/man/extra/arch-install-scripts/genfstab.8>.
+
+- Generate the output based on LABEL:
+
+`genfstab -L {{path/to/mount_point}}`
+
+- Display `fstab` output using partitions' UUID:
+
+`genfstab -U {{path/to/mount_point}}`
+
+- Generate a fstab file, requires being a superuser:
```

Comment:
Instead of giving superuser notice, you can just add `sudo` to the command (see additional suggestion below):
```suggestion
- Generate an fstab file:
```

Reply:
An fstab? I do not understand the edit.

Reply:
"an fstab file".  I assume "fstab" is generally pronounced "ef-es-tab"&mdash;@navarroaxel am I correct about this?

Reply:
It's better English to use "an" before a word like "fstab", the way it's generally pronounced.  Now of course, people have different pronunciations but I believe "an" would be a better fit here.
```suggestion
- Generate an fstab file, requires being a superuser:
```
Please also disregard the first suggestion, as I understand the superuser situation now :)

Reply:
Yeah, that's correct grammar there @bl-ue.